.github/workflows/income-limits-data-sync.yml

```diff
@@ -100,4 +100,4 @@ jobs:
 
       - name: Copy files to the S3 bucket
         run: |
-          aws s3 sync ${{ env.TEMP_FOLDER }} s3://${{ env.BUCKET_NAME }}
+          aws s3 sync ${{ env.TEMP_FOLDER }} s3://${{ env.BUCKET_NAME }} --acl public-read
```

Comment:
Ensures that the CSVs are uploaded with public read rights. The entire bucket is public, but this ensures that the files themselves are public as well.